src/game/objects/har.c

```diff
@@ -148,6 +148,14 @@ void har_event_hazard_hit(har *h, bk_info *info) {
     fire_hooks(h, event);
 }
 
+void har_event_enemy_hazard_hit(har *h) {
+    har_event event;
```

Comment:
hmmh, this is a bit unsafe memory use .. should memset the sctruct to zero always before use. However, it looks this is always used like this in the file. Could a commit be added that memsets all such har_events to zero before passing it on to `fire_hooks()`

Reply:
Hmm. Could we create an issue to specifically address this in all of the events? That way we can keep the scope of the changes in this ticket down.